packages/frontend/src/components/Explorer.tsx

```diff
@@ -551,7 +569,25 @@ export const Explorer: FC<Props> = ({ savedQueryUuid }) => {
                 title={'Delete'}
             >
                 <div className={Classes.DIALOG_BODY}>
-                    <p>Are you sure you want to delete this chart ?</p>
+                    <p>
+                        Are you sure you want to delete the chart{' '}
+                        <b>"{chartName}"</b> ?
+                    </p>
+
+                    {relatedDashboards && relatedDashboards.length > 0 && (
+                        <>
+                            <b>
+                                This action will affect{' '}
```

Comment:
@PriPatel @rephus - we could be more specific about the consequences:

> This action will remove a chart tile from X dashboards:

Reply:
I like it! Small thing - do the tiles actually get removed? 
Maybe that doesn't matter! It gets the point across a lot more clearly. Lets go for it !

Reply:
Done 
![image](https://user-images.githubusercontent.com/1983672/162160287-78d04db0-7e47-4eeb-8951-ee974321e922.png)

Reply:
Good point, it sets them as missing (although there was a discussion about actually deleting them). How about

> This will remove the chart from X tiles on dashboards: